public/keymaps/nafuda_default .json

```
@@ -0,0 +1,28 @@
+{
+  "keyboard": "naked60/rev1",
+  "keymap": "default_ad36bb3",
+  "commit": "687a24f2980e6e490ac8e43104f4176b033c9243",
+  "layout": "LAYOUT",
+  "layers": [
+    [
+                          "TG(1)",
+"LT(3, KC_BSPC)",         "KC_UP",     "TG(2)",
+       "KC_LEFT",       "KC_DOWN",   "KC_RIGHT"
```

Configurator almost entirely uses the seven-character-maximum keycodes for its polished rendering mode.

```suggestion
       "KC_LEFT",       "KC_DOWN",   "KC_RGHT"
```